src/chat/FetchError.js

```diff
@@ -28,7 +29,13 @@ export default class FetchError extends PureComponent<Props> {
   render() {
     return (
       <View style={styles.container}>
-        <Label style={styles.text} text="Oops! Something went wrong." />
+        {(() => {
+          if (this.props.error instanceof TimeoutError) {
+            return <Label style={styles.text} text="Request timed out." />;
+          } else {
+            return <Label style={styles.text} text="Oops! Something went wrong." />;
+          }
+        })()}
```

Comment:
Probably simpler as a `… ? … : …` expression, no?

Reply:
> Probably simpler as a `… ? … : …` expression, no?

Sure. I used `if` / `else` because I thought there might be other cases to consider in the future, and this form would make it easier to expand on.

Reply:
I see. Yeah, that seems like a fine reason.